src/components/Person/Person.jsx

```diff
@@ -1 +1,9 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    <p className="Person__age">{age ? (`I am ${age}`) : null}</p>
```

Comment:
Let's refactor it. You don't need to use a ternary operator
```suggestion
   {age && <p className="Person__age">{`I am ${age}`}</p>}
```